openssl/src/ssl/mod.rs

```diff
@@ -3104,6 +3104,19 @@ impl SslRef {
             }
         }
     }
+    #[corresponds(SSL_add1_chain_cert)]
+    #[cfg(ossl102)]
+    pub fn add_chain_certificate_pem(&mut self, chain: &[u8]) -> Result<(), ErrorStack> {
+        let cert = X509::from_pem(chain)?;
+        let ret = unsafe {
+            ffi::SSL_add_chain_certificate_pem(self.as_ptr(), cert.as_ptr() as *mut _ as *mut _)
```

Comment:
:thinking: in `as *mut _ as *mut _` isn't one `as *mut _` redundant?

Reply:
thankyou, removed :)